tests/testsuite/build.rs

```diff
@@ -2384,6 +2386,112 @@ fn rebuild_preserves_out_dir() {
         .run();
 }
 
+#[test]
+fn rebuild_on_mid_build_file_modification() {
```

Comment:
I would put `rebuild_on_mid_build_file_modification` in `freshness.rs`. That is where most of the fingerprint tests are located, and `build.rs` is crazy long.

Reply:
I will move it.